sdk/src/main/java/com/mapbox/maps/ViewAnnotationManagerImpl.kt

```diff
@@ -119,14 +120,16 @@ internal class ViewAnnotationManagerImpl(
   }
 
   private fun validateOptions(options: ViewAnnotationOptions) {
-    checkNotNull(options.geometry) { EXCEPTION_TEXT_GEOMETRY_IS_NULL }
+    if (options.geometry == null) {
```

Comment:
nit:
what about custom extension `checkNotNull(condition, exceptionTYPE, message)`?

Reply:
why do we need it? seems like an overkill to me